### DISCUSSION TITLE
Add a reporting system. Add Discord Integration - Like in DiscordSrv

### DISCUSSION THREAD

**Discussion by AugustDeplazza1** [Zrips/CMI#7179] [Enhancement]
What would be the console, sending reports, synchronizing roles / nicknames, make a white list of roles that will be sent to the chat from the discord because the discordsrv does not provide that if there is a color in the roles that JuniperBot adds, then when you write in the ds channel and go to minecraft, then writes instead of Ovner or Admin color (JB-CLR-#FF0F17).

**Comment by mrfloris** [comment]
Interesting, yeh. 

I now use /helpop and /staffmsg in a customalias /report command, that also uses discordsrv's alert.yml to send an embed msg with the report to a private #reports channel.
